packages/desktop/tracker.js

```diff
@@ -38,6 +39,40 @@ export class ActiveWindowWatcher {
 
 		const data = appDataObject(startTime, endTime);
 
+		// Making a post request to '/store'
+		const body = JSON.stringify({
+            'name': name,
+            'title': title,
+            'startTime': startTime,
+			'endTime': endTime
+        });
+
+        const post_to = {
+            host: '127.0.0.1',
+            port: '3000',
+            path: '/store',
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'Content-Length': body.length
+            }
+        };
+
+        const req = http.request(post_to, res => {
+            console.log(`statusCode: ${res.statusCode}`)
+
+            res.on('data', d => {
+                process.stdout.write(d);
+            })
+        })
+
+        req.on('error', error => {
+            console.error(error);
+        })
+
+        req.write(body);
+        req.end();
```

Comment:
That looks painful. Maybe we can use [axios](https://axios-http.com/).

Reply:
+1